extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -441,6 +471,10 @@ int64_t ListenHTTP::WriteCallback::process(std::shared_ptr<io::BaseStream> strea
   return nlen;
 }
 
+std::string ListenHTTP::getPort() const {
+  return listeningPort;
```

Comment:
Why is a port a string? Is it because ports are appended with s?

Reply:
Yes.